### ISSUE TITLE
Implement semantics for `tasklist` metadata on experiment folder

### ISSUE THREAD

**Issue by waxlamp** [arclamp/annotation-tracker#7]
This will be the way to store the experimental setup structure with a folder of images to be used in that experiment.

The `tasklist` metadatum may be a JSON list of strings (naming the tasks) or a list of objects (containing structured data about each task, including the name).

**Comment by BryonLewis** [comment]
I structured it as experiments containing numerous tasks because the current structure of the code already had that in there.  It can easily be changed afterwards to more of a flat tasklist -list as well as more structured metadata for display and a more concrete transition between tasks/images.


https://user-images.githubusercontent.com/61746913/108763871-e2890a80-751f-11eb-802f-5ef1fecf1b21.mp4

Just figured the video would explain what I had added to the existing code.  I just took a quick look at your other tasks and am thinking of ways to build on the current stuff to add them in.
